Source/JavaScriptCore/llint/LLIntOfflineAsmConfig.h

```diff
@@ -188,6 +188,12 @@
 #define OFFLINE_ASM_JIT 0
 #endif
 
+#if ENABLE(JIT_CAGE)
+#define OFFLINE_ASM_JIT_CAGE 1
```

Comment:
Where is this used? The commit message says "below", but I don't see it anywhere else in the diff.

Reply:
offlineasm has ifdef-like conditional compilation statements, which use tokens that expand to `OFFLINE_ASM_`-prefixed macros in generated code. So `if JIT_CAGE` here is the use site:

https://github.com/WebKit/WebKit/blob/9aa6cb6f001fbedca9258e81d0b3a6858f7040f7/Source/JavaScriptCore/llint/LowLevelInterpreter.asm#L1847-L1854

Reply:
offlineasm has ifdef-like conditional compilation statements, which use tokens that expand to `OFFLINE_ASM_`-prefixed macros in generated code. So `if JIT_CAGE` here is the use site:

https://github.com/WebKit/WebKit/blob/9aa6cb6f001fbedca9258e81d0b3a6858f7040f7/Source/JavaScriptCore/llint/LowLevelInterpreter.asm#L1847-L1854